CONTRIBUTING.rst

```diff
@@ -62,7 +62,7 @@ On Ubuntu you need these:
 .. code:: sh
 
     sudo apt install texlive-base texlive-latex-extra texlive-fonts-extra python3-matplotlib graphviz
-    pip3 install pelican Pyphen Pillow
+    pip3 install pelican Pyphen pillow
```

Comment:
Interesting, [here it says uppercase](https://pypi.org/project/Pillow/)?

Reply:
Er... I dunno. I had to use a lowercase P for this to work

Reply:
package squatting in action? (are you involuntarily part of some rogue botnet now?) 

didn't even know case mattered for pip

Reply:
guess I'm inadvertently a crypto miner now